File: 프로그래머스/unrated/181874. A 강조하기/A 강조하기.java
import java.util.*;

class Solution {
    public String solution(String myString) {
        String answer = "";
        char[] myChar = myString.toCharArray();
        for(int i=0;i<myChar.length;i++){
            if(myChar[i] == 'a') myChar[i] = 'A';
            else if(myChar[i] == 'A') continue;
            else myChar[i] = Character.toLowerCase(myChar[i]);
        }
        return String.valueOf(myChar);
    }
}

File: 프로그래머스/unrated/181874. A 강조하기/A 강조하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string myString) {
    for(auto &c : myString){
        if(c == 'a')
            c = 'A';
        else if(c == 'A') continue;
        else
            c = tolower(c);
    }
    return myString;
}